content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -45,6 +45,7 @@ Refer to the following table for Windows operating system support in Kubernetes.
 | *Kubernetes v1.17* | Windows Server 2019 | Windows Server ver 1809 |
 | *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1809, Windows Server ver 1903, Windows Server ver 1909 |
 | *Kubernetes v1.19* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
+| *Kubernetes v1.20* | Windows Server 2019 | Windows Server ver 1909, Windows Server ver 2004 |
```

Comment:
Should we start to drop off the 1.14/1.15 releases from this list to avoid confusion on what k8s versions are supported?

Reply:
agreed., we should also remove 1.16, right? since this will go out with 1.20 release

Reply:
Sounds good.